source/deployment-options/docker/wazuh-container.rst

```diff
@@ -260,7 +260,7 @@ To improve security, you can change the default password of the Wazuh users. The
 Wazuh indexer users
 ~~~~~~~~~~~~~~~~~~~
 
- To change the password of the default ``admin`` and ``kibanaserver`` users, do the following.
+ To change the password of the default ``admin`` and ``kibanaserver`` users, do the following. They can only be changed one at a time. 
```

Comment:
```suggestion
 To change the password of the default ``admin`` and ``kibanaserver`` users, do the following. You can only change one at a time. 
```

Reply:
Done in https://github.com/wazuh/wazuh-documentation/pull/6554/commits/ec962baf1eaa5d64ec7831158695470b4c101785.